client/debug.log

```diff
@@ -0,0 +1,15 @@
+[0726/202656.338:ERROR:crash_report_database_win.cc(428)] unexpected header
```

Comment:
Remember that log files should never be pushed since they may contain important information.